### ISSUE TITLE
Recognize OpenTelemetry Severity Levels

### ISSUE THREAD

**Issue by djaglowski** [observIQ/stanza#180]
See [this issue](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/1420) for more context.

In short, the default mapping should be updated to include values such as `info2`, `info3`, `info4`, `debug2`,`debug3`,`debug4`, etc.

**Comment by djaglowski** [comment]
In addition to recognizing these levels, we should also formally add them to our enumeration. It will be necessary to distinguish between them on a numeric-only basis.

```
Error Severity = 60
Error2 Severity = 62
Error3 Severity = 63
Error4 Severity = 64
```

This makes the stanza severity system a superset of the OT system, which allows for full service of OT use cases while also supporting all existing stanza mappings.

**Comment by djaglowski** [comment]
Addressed by #188
